KerbalStuff/config.py

```diff
@@ -15,13 +15,27 @@ def get_env_var_or_config(section, key):
     if env_var:
         return env_var
     else:
-        return config.get(section, key)
+        return config.get(section, key, fallback=None)
 
 
-_cfg = lambda k: get_env_var_or_config(env, k)
-_cfgi = lambda k: int(_cfg(k))
-_cfgb = lambda k: strtobool(_cfg(k)) == 1
-_cfgl = lambda k: ast.literal_eval(_cfg(k))
+def _cfg(k):
+    return get_env_var_or_config(env, k)
+
+
+def _cfgi(k):
+    val = _cfg(k)
+    return int(val) if val is not None else 0
```

Comment:
This would handle non-int strings as well as missing values.
And I'm suggesting to always return `None` so that the code upstream could distinguish _missing_ values from actual ones.
```suggestion
    try:
        return int(val)
    except (ValueError, TypeError):
        return None
```
Also, if we're in debug mode, we could log the error here in except block.

Reply:
If non-int strings are set for an int setting, there **should** be an exception thrown. That represents an error by the site admins, they should know about it.

The point of this function is to return a setting as an int. `None` would mean special handling code all over the place to avoid crashing. 0 is a sensible default for an int.

Reply:
We could also use -1 as the default for integers.

Reply:
You could also use `isnumber(val)` to prevent exceptions.